.golangci.yml

```diff
@@ -0,0 +1,8 @@
+# all available settings of specific linters
+linters-settings:
+  gocyclo:
+    # minimal code complexity to report, 30 by default (but we recommend 10-20)
+    min-complexity: 50
```

Comment:
The recommendation for the complexity value is 10-20. Could you explain why you choose 50?

Reply:
The porting Ethereum codes makes complexity high, but I think it is hard to reduce them. This is the easiest way to pass the linter test without the complexity. Any suggestion?

Reply:
I was just curious. Thanks for the comment!